### ISSUE TITLE
systemd-id128 invocation-id always returns the user manager invocation id

### ISSUE THREAD

**Issue by fsateler** [systemd/systemd#11649]
**systemd version the issue has been seen with**
> 240

**Used distribution**
 > Debian

**Expected behaviour you didn't see**
 
```
% systemd-run --user --pipe systemd-id128 invocation-id
Running as unit: run-u132649.service
e9fd300dbcbe4fc59541d72d53ecbc04
% systemd-run --user --pipe systemd-id128 invocation-id
Running as unit: run-u132651.service
95130fcaf9664d969b01ce32e6b2f3a0
% journalctl INVOCATION_ID=95130fcaf9664d969b01ce32e6b2f3a0
-- Logs begin at Wed 2018-08-15 22:53:55 -03, end at Mon 2019-02-04 15:05:05 -03. --
Feb 04 15:03:44 felipedell systemd[1]: Started /usr/bin/systemd-id128 invocation-id.
Feb 04 15:03:45 felipedell systemd[1]: run-u132651.service: Succeeded.
```

**Unexpected behaviour you saw**

```
% systemd-run --user --pipe systemd-id128 invocation-id     
Running as unit: run-u371.service
05159fb4a25046f885c00cdd8207cfc7
% systemd-run --user --pipe systemd-id128 invocation-id
Running as unit: run-u372.service
05159fb4a25046f885c00cdd8207cfc7
% journalctl INVOCATION_ID=05159fb4a25046f885c00cdd8207cfc7
-- Logs begin at Wed 2018-08-15 22:53:55 -03, end at Mon 2019-02-04 15:04:58 -03. --
Jan 31 18:07:34 felipedell systemd[1]: Starting User Manager for UID 1000...
Jan 31 18:07:34 felipedell systemd[1]: Started User Manager for UID 1000.
```

**Comment by yuwata** [comment]
I've proposed #11658. PTAL.

**Comment by yuwata** [comment]
I tentatively drop 'has-pr' tag to encourage other people to create PRs for this issue :-)

**Comment by poettering** [comment]
Fix waiting in #11991

**Comment by yuwata** [comment]
Sorry, I forgot to drop 'Closes #11649.' comment in #11658.
